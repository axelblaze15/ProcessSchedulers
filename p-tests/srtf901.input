10000000000000 2000 1
1000 8 1
1000000000 2000 1
1000000 200 100000
1000000000000000000 2000 1
10 4 1