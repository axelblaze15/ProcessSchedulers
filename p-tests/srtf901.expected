10 5 0 1
1000 9 0 1
1000000 303 3 103
1000000000 251 0 1
10000000000000 252 1 2
1000000000000000000 253 2 3
1 19